web-services/query/src/main/java/datawave/webservice/query/logic/BaseQueryLogic.java

```diff
@@ -77,11 +72,13 @@ public BaseQueryLogic(BaseQueryLogic<T> other) {
         setConnPoolName(other.getConnPoolName());
         setPrincipal(other.getPrincipal());
         setRoleManager(other.getRoleManager());
-        setMarkingFunctions(other.getMarkingFunctions());
-        setResponseObjectFactory(other.getResponseObjectFactory());
         setSelectorExtractor(other.getSelectorExtractor());
     }
     
+    public GenericQueryConfiguration getConfig() {
+        return baseConfig;
```

Comment:
Perhaps we should lazily create the baseConfig so that we at least get a NPE if somebody tries to access it directly but it is not the primary configuration.

Reply:
another option here is to remove baseConfig entirely, and make getConfig abstract. I think I like that better